spec/mumukit/write_spec.rb

```diff
@@ -3,203 +3,248 @@
 describe Mumukit::Sync::Store::Github::GuideWriter do
 
   let(:bot) { Mumukit::Sunc::Store::Github::Bot.new('mumukibot', 'zaraza') }
-
-  let!(:exercise_1) { guide[:exercises].first }
```

Comment:
Although indentation changed, I only added a new export scenario, and tested that both new fields and the guide expectations are being written.